package.json

```diff
@@ -32,6 +34,7 @@
     "format": "prettier --loglevel warn --write \"**/*.{ts,tsx,css,md}\""
   },
   "dependencies": {
+    "@react-aria/utils": "^3.8.0",
```

Comment:
These two new dependencies were ones that we used, but didn't have in the package.json, so they weren't correctly marked as external.